app/manifest.json

```diff
@@ -156,6 +156,15 @@
     },
     "45-disable": {
       "description": "UTILITY: Do nothing (disable browser shortcut - experimental)"
+    },
+    "46-opensettings": {
+      "description": "LOCATION: Open Settings Page"
+    },
+    "47-openextensions": {
+      "description": "SCROLLING: Open Extensions Page"
```

Comment:
Should probably say LOCATION instead of SCROLLING, I think?

Reply:
Ah yes, I must have been in a hurry, sorry.